app/views/ProjectsView/components/AddEditProjectDialog/index.tsx

```diff
@@ -105,25 +104,16 @@ const AddEditProjectDialog = ({
     }
   }, [editData]);
 
-  const handleClose = useCallback(async () => {
-    if (projectName.length) {
-      const newEntry = {
-        name: projectName,
-        description: projectDesc,
-      };
-
+  const projectMutation = useMutation(
```

Comment:
can destructure like so
`const { isLoading: isSaving, mutate: mutateProject } = useMutation(` 
instead of changing more places 

we try to use more destructuring instead of `objectA.methodB`, it's to make things more readable